src/contentmanagerside.cpp

```diff
@@ -92,7 +93,14 @@ void ContentManagerSide::setContentManager(ContentManager *contentManager)
 
 void ContentManagerSide::setCategories(QStringList categories)
 {
-    mp_categories->setSelections(categories, KiwixApp::instance()->getSettingsManager()->getCategoryList());
+    const auto lang = QLocale().name().split('_')[0].toStdString();
+    KiwixChoiceBox::SelectionList sList;
+    for (const auto& category : categories) {
+        const auto translation = QString::fromStdString(kiwix::translateBookCategory(lang, category.toStdString()));
+        sList.append({category, translation});
```

Comment:
This seems to be inconsistent with the next commit. Here the order of the pair is `(identifier, translation)` whereas in `SettingsManager` it appears to be `(translation, identifier)`

Reply:
SelectionList and Translation list does have things flipped. However, I believe we should do it in a seperate place as a part of refactoring the settingsmanager part of the side bar. It would be a redundant change if we decide to only store the key in the future.

Reply:
Moved to #1185